### ISSUE TITLE
Hero(Media Blend) - Part 2

### ISSUE THREAD

**Issue by sdmcraft** [urfuwo/hlx-test#135]
Items pending after #134 

5. `udex-avatar` web component
6. `udex-button` web component
7. Author page link
8. Reading time
9. Intro text

Related links:
1. [Udex component](https://udextest.z6.web.core.windows.net/285/index.html?path=/story/patterns-containers-hero-banner--media-blend&globals=theme:Horizon%20Brand) 
2. [Figma (For Adobe)](https://www.figma.com/file/oSetT4LbatRmXlcB2A7V8Z/Content-Hubs-2024?type=design&node-id=268-3637&mode=design&t=x6IEAdpxr7CBvLbK-0)
3. [Figma (Brand Digital Design)](https://www.figma.com/file/OFEfvhyXJvYJ6MQZ30699m/v2.0-Brand-Digital-Design-System?type=design&node-id=10175-2818&mode=design&t=sg7sletrzPVv0nbt-0)

## `udex-avatar` web component, `udex-button` web component

check if now available in UDEx and if yes, include based on that implementation
OK to use the EDS avatar rather than the UDEx avatar, as that is what is already done in other places.

## Author page link, Reading time, Intro text

implementation missing in EDS block
Reading time might need adding to the index - might already show up if available in metadata (check)
